File: Fines2Model3/Values/Plan.cs
using SqlOrganize.ValueTypesUtils;

namespace SqlOrganize.Sql.Fines2Model3      
{
    public class PlanValues : EntityVal
    {
        public PlanValues(Db _db, string entityName, string? _fieldId = null) : base(_db, entityName, _fieldId)
        {
        }

        public override string ToString()
        {
            string s = "";
            s += GetStr("orientacion", "?")?.Acronym();
            s += " ";
            s += GetStr("resolucion", "?");
            return s;
        }
    }
}

File: src/Fines2/Model/Plan_.php
<?php

namespace Fines2\Model;

use \Fines2\Model\Plan;

use SqlOrganize\Sql\Entity;
use SqlOrganize\Utils\ValueTypesUtils;

use Exception;
use DateTime;

class Plan_ extends Plan
{
public function getLabel(): string{
        return ValueTypesUtils::acronym($this->orientacion ?? "") . " " . ($this->resolucion ?? "?"); 
    }
}

